keadm/cmd/keadm/app/cmd/util/set.go

```diff
@@ -0,0 +1,501 @@
+	switch parseType(vals[0]) {
+	case "int":
+		intArray := make([]int, len(vals))
+		for i, v := range vals {
+			v = strings.TrimSpace(v)
+			intValue, _ := strconv.Atoi(v)
+			intArray[i] = intValue
+		}
+		return intArray
+	case "float":
+		floatArray := make([]float64, len(vals))
+		for i, v := range vals {
+			v = strings.TrimSpace(v)
+			floatValue, _ := strconv.ParseFloat(v, 64)
+			floatArray[i] = floatValue
+		}
+		return floatArray
+	default:
+		stringArray := make([]string, len(vals))
+		for i, v := range vals {
+			stringArray[i] = strings.TrimSpace(v)
+		}
+		return stringArray
+	}
+}
+
+// ParseType parses the type of array and interprets it to int, float64, string.
+func parseType(s string) string {
+	// Check if it's an integer
+	if _, err := strconv.Atoi(s); err == nil {
+		return "int"
+	}
+	// Check if it's a float
+	if _, err := strconv.ParseFloat(s, 64); err == nil {
+		return "float"
+	}
+	// Check if it's a bool
+	if _, err := strconv.ParseBool(s); err == nil {
+		return "bool"
+	}
+	// Otherwise, it's a string
+	return "string"
+}
+
+// GetNameFormStatus judges the types of names.
+// The name has three forms:
+//
+//	1)name1 = value1, return 0
+//	2)name1[0].variable1 = value1, return 1
+//	3)name1[0] = value1, return 2
+func getNameFormStatus(s string) int {
+	if !strings.Contains(s, "[") && !strings.Contains(s, "]") {
+		return 0
+	}
+	closeBracketIndex := strings.Index(s, "]")
+	if closeBracketIndex != (len(s) - 1) {
+		return 1
+	}
+	if closeBracketIndex == (len(s) - 1) {
+		return 2
+	}
+	return -1
+}
+
+// SetCommonValue modifies the new value of the name in the config represented by struct.
+// The type of new value may be int, float, string, splic.
+// The name is represented by name1.name2.(...).nameM.
+func setCommonValue(structPtr interface{}, fieldPath string, value interface{}) error {
+	structVal := reflect.ValueOf(structPtr).Elem()
+
+	path := strings.Split(fieldPath, ".")
+	var fieldVal reflect.Value
+	fieldVal = structVal
+	for _, p := range path {
+		fieldVal = getFieldValue(fieldVal, p)
+		if !fieldVal.IsValid() {
+			return fmt.Errorf("%s: No such field: %s in Config", fieldPath, p)
+		}
+	}
+
+	val := reflect.ValueOf(value)
+
+	if fieldVal.Type() != val.Type() {
+		return fmt.Errorf("%s: Provided value type %s does not match field type %s", fieldPath, val.Type(), fieldVal.Type())
+	}
+	fieldVal.Set(val)
+
+	return nil
+}
+
+// GetFieldValue gets the fieldname of the struct.
+func getFieldValue(v reflect.Value, fieldName string) reflect.Value {
+	if v.Kind() == reflect.Ptr {
+		v = v.Elem()
+	}
+
+	f := v.FieldByName(fieldName)
+	if !f.IsValid() {
+		return reflect.Value{}
+	}
+	return f
+}
+
+// ParseAndSetArrayValue combines the function of SetArrayValue and ParseFieldPath1.
+func parseAndSetArrayValue(structPtr interface{}, fieldPath string, newValue interface{}) error {
+	path, index, _ := parseFieldPath1(fieldPath)
+	err := setArrayValue(structPtr, path, index, newValue)
+	// fmt.Println(newValue)
```

Comment:
If this is used for debugging, please remove it from the official code. If it is necessary output, please use klog.info to output the log.

Reply:
OK,I have removed these code.